codefordenver/code-for-denver-site
# restructure to run on Jekyll

Takes the existing directory structure, slightly modifies it and peppers in a few new files and directories in order to get the app to run on [jekyll](http://jekyllrb.com/).

##### To test
- ```gem install jekyll```
- at project root: ```jekyll serve --baseurl '' --watch```
- visit ```http://localhost:4000/``` and make sure all is well
